### PR TITLE
Support JDK 11

### PR DESCRIPTION
**Pull request by frenchy64**
> Close https://github.com/threatgrid/iroh/issues/3132

<a name="qa">[§](#qa)</a> QA
============================

Please test all functionality. This commit upgrades many dependencies which may have unexpected results beyond our unit tests.

<a name="release-notes">[§](#release-notes)</a> Release Notes
=============================================================

```
intern: update dependencies and test JDK 11
```

<a name="squashed-commits">[§](#squashed-commits)</a> Squashed Commits
======================================================================

## Notes:

Depends on:
- https://github.com/threatgrid/clj-momo/pull/67
- https://github.com/threatgrid/ctim/pull/269

See also:
- https://github.com/threatgrid/iroh/pull/3143 (uses above PR's)

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -27,122 +32,94 @@
...
-                               org.clojure/clojurescript]]
-                 [threatgrid/clj-momo "0.3.3"]
+                 [prismatic/schema "1.1.12"]
+                 [metosin/schema-tools "0.12.2"]
+                 [threatgrid/flanders "0.1.23-20200204.180031-3"] ;sha: f92c65bb811e97cc0b96443e46474ba8caa1e1cc
+
+                  
+                 [threatgrid/ctim "1.0.16-20200205.182529-5"] ;sha: 121b661f9638d6720c2ac22f2bdd125383abdc8a
+                 [threatgrid/clj-momo "0.3.4-20200204.172456-2"] ;sha: 296203d2bc07e0063103e4bc1cc0c921284db927
+
```
Comment (msprunck) [inline project.clj]: Don't forget to use the released version of `flanders`, `ctim` and `cli-momo` before merging

File: project.clj [hunk 2]
```diff
@@ -27,122 +32,94 @@
...
-                 [threatgrid/ring-turnstile-middleware "0.1.0"
-                  :exclusions [metosin/schema-tools]]
-                 [threatgrid/ring-jwt-middleware "1.0.0"
-                  :exclusions [metosin/compojure-api]]
-                 [scopula "0.1.4"
-                  :exclusions [org.clojure/spec.alpha
-                               org.clojure/clojure
-                               org.clojure/core.specs.alpha
-                               com.andrewmcveigh/cljs-time]]
+                 [threatgrid/ring-turnstile-middleware "0.1.1-20200203.182733-1"];sha: 6bef68919a0038d55721c44a84c03e7827a26f42
```
Comment (msprunck) [inline project.clj]: same here

File: project.clj (hunk 1 above)
Comment (frenchy64) [reply project.clj]: Bumped flanders, ctim, clj-momo.

File: project.clj (hunk 2 above)
Comment (frenchy64) [reply project.clj]: Using threatgrid/ring-turnstile-middleware "0.1.1".

**Comment by hshahman** [comment]
Verified in rel-v1.43

**Pull request merged**